kuma/users/utils.py

```diff
@@ -46,14 +46,21 @@ def retrieve_stripe_subscription_info(user):
     )
     if stripe_subscription:
         source = stripe_customer.default_source
+        if source.object == "card":
+            card = source
+        elif source.object == "source":
+            card = source.card
+        else:
+            return None
```

Comment:
Why this? 
I think I'd rather learn what that could be. E.g....
```suggestion
            raise NotImplementedError(f"don't know how to deal with {source.object!r}")
```

Reply:
Hopefully, that'll either raise an error, which we learn from in Sentry, or it never happens.

Reply:
You're right, we should raise something here. I went with `ValueError`.